src/components/notifications/GtxMessaging.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+    <div class="mb-3">
+        <label for="gtxmessaging-api-key" class="form-label">{{ $t("gtxMessagingApiKey") }}</label>
+        <HiddenInput id="gtxmessaging-api-key" v-model="$parent.notification.gtxMessagingApiKey" :required="true"></HiddenInput>
+    </div>
+    <div class="mb-3">
+        <label for="gtxmessaging-from" class="form-label">{{ $t("gtxMessagingFrom") }}</label>
+        <input id="gtxmessaging-from" v-model="$parent.notification.gtxMessagingFrom" type="text" class="form-control" required>
+    </div>
+    <div class="mb-3">
+        <label for="gtxmessaging-to" class="form-label">{{ $t("gtxMessagingTo") }}</label>
+        <input id="gtxmessaging-to" v-model="$parent.notification.gtxMessagingTo" type="text" pattern="^[\d-]{10,31}(@[\w\.]{1,})?$" class="form-control" required>
+    </div>
+    <div class="mb-3">
+        <i18n-t tag="p" keypath="More info on:" style="margin-top: 8px;">
+            <a href="https://www.gtx-messaging.com/en/solutions/sms-gateway-api/" target="_blank">https://www.gtx-messaging.com/en/solutions/sms-gateway-api/</a>
+        </i18n-t>
+    </div>
```

Comment:
A few things here:
- please don't use `margin-top` => use the `mt-XXX` classes instead 
- I think this can be simplified into a single i18n-t tag => removing the outer `div`
- The site you are linking to is a marketing page. What is the intend here? (what info are you trying to convey?)

Reply:
I took this section from the Twilio provider and linked to the respective section in gtx. Happy to remove this part. Please advise.

Reply:
Depends, what this tries to add.
If this adds additional context, then it should be kept.
Just linking to a marketing page is not as usefull imo, as those are usually not designed to give context.

What context is the link trying to convey?
If it is just for completeness, this can be removed, as the helptexts copy the relevant part of the documentation.
If there are parts of the docs which are too long/change frequently/... they should be linked.

Reply:
I studied other notification providers and aligned mine with the style found in the others.
E.g.
- https://github.com/louislam/uptime-kuma/blob/master/src/components/notifications/Bark.vue#L14C23-L14C51
- https://github.com/louislam/uptime-kuma/blob/master/src/components/notifications/FlashDuty.vue#L6C22-L6C62
- https://github.com/louislam/uptime-kuma/blob/master/src/components/notifications/GoAlert.vue#L6C21-L6C40
- https://github.com/louislam/uptime-kuma/blob/master/src/components/notifications/Apprise.vue#L19C26-L19C59

I personally don't need it. I think, people will add providers to kuma they are already familiar with. Given the hints on the input fields this should be enough. I will remove this section